lib/screens/questionaire_screen/widgets/cover.dart

```diff
@@ -18,80 +18,109 @@
 import 'package:flutter/material.dart';
 
 import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:picos/widgets/picos_body.dart';
 
 import '../../../themes/global_theme.dart';
 import '../../../widgets/picos_add_button_bar.dart';
-import '../../../widgets/picos_body.dart';
 import '../../../widgets/picos_ink_well_button.dart';
 
 /// A cover for the different sections of the questionaire.
 class Cover extends StatelessWidget {
   /// Constructs the cover page.
   const Cover({
     this.title = '',
+    this.image = '',
     Key? key,
     this.backFunction,
     this.nextFunction,
   }) : super(key: key);
 
   /// The title cover.
   final String title;
+
+  /// Contains the annotation for the image.
+  final String image;
+
   /// Function for getting a page back.
   final void Function()? backFunction;
+
   /// Function for getting the next page.
   final void Function()? nextFunction;
 
   @override
   Widget build(BuildContext context) {
     final String back = AppLocalizations.of(context)!.back;
     final String next = AppLocalizations.of(context)!.next;
+    final String start = AppLocalizations.of(context)!.letsStart;
     final GlobalTheme theme = Theme.of(context).extension<GlobalTheme>()!;
-    final double height = MediaQuery.of(context).size.height;
+    final double height = MediaQuery.of(context).size.height -
+        kToolbarHeight -
+        MediaQuery.of(context).viewPadding.top;
 
-    return PicosBody(
-      child: Column(
-        children: <Widget>[
-          SizedBox(
-            height: height / 3,
-          ),
-          Center(
-            child: Text(
-              title,
-              textAlign: TextAlign.center,
-              style: const TextStyle(
-                fontSize: 40,
-              ),
+    return Container(
+      height: height,
+      color: theme.darkGreen1,
+      child: PicosBody(
+        child: Column(
+          children: <Widget>[
+            SizedBox(
+              height: height / 5,
             ),
-          ),
-          SizedBox(
-            height: height / 3.5,
-          ),
-          PicosAddButtonBar(
-            shadows: false,
-            leftButton: PicosInkWellButton(
-              padding: const EdgeInsets.only(
-                left: 30,
-                right: 13,
-                top: 15,
-                bottom: 10,
+            Center(
+              child: Image(
+                image: AssetImage(image),
               ),
-              text: back,
-              onTap: backFunction ?? () {},
-              buttonColor1: theme.grey3,
-              buttonColor2: theme.grey1,
             ),
-            rightButton: PicosInkWellButton(
-              padding: const EdgeInsets.only(
-                right: 30,
-                left: 13,
-                top: 15,
-                bottom: 10,
+            Center(
```

Comment:
These center widgets might be redundant.
When testing I could remove them and the elements are still centered.
Do they have any other purpose?